vcr/interface.go

```diff
@@ -40,11 +42,7 @@ type Finder interface {
 }
 
 // Writer is the interface that groups al the VC write methods
-type Writer interface {
-	// StoreCredential writes a VC to storage. Before writing, it calls Verify!
-	// It can handle duplicates.
-	StoreCredential(vc vc.VerifiableCredential, validAt *time.Time) error
-}
+type Writer = types.Writer
```

Comment:
The problem is the VCR Writer writes to the VCR Store, which is a collection of all issued and credentials present on the node (issued to you privately/publicly, and issued to others, but publicly). So we need to write it from the `issuer` and `holder` packages, but can't use it from the main `vcr` package, since that would introduce cyclic dependencies.